### PR TITLE
build: adding yaml pre-commit check

### PR DESCRIPTION
**Pull request by Margar1ta**
YAML pre-commit check on changed files is added with the use of lint-staged and pre-commit packages.
If a mistake is found git will show:
myakovleva-2020-02-27_16-57-45

### CODE REVIEW

**Comment by PositiveJS** [comment]
Preview docs changes for 8f55a08 at https://positive-js.github.io/mosaic-previews/pr419-8f55a08/

File: package.json [hunk 1]
```diff
@@ -211,6 +214,10 @@
         "server-dev:validation": "npm run server-dev -- --env.component validation",
         "server-dev:vertical-navbar": "npm run server-dev -- --env.component vertical-navbar"
     },
+    "lint-staged": {
+        "*.yml": "yamllint"
+    },
+    "pre-commit": "lint-staged",
```
Comment (Fost) [inline package.json]: we already have `husky` for create git hooks.

https://github.com/typicode/husky#install

**Comment by Margar1ta** [comment]
`husky` worked perfectly:
myakovleva-2020-02-27_19-18-35

**Comment by PositiveJS** [comment]
Preview docs changes for db46786 at https://positive-js.github.io/mosaic-previews/pr419-db46786/

**Pull request merged**